recipes/lifetimes/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "Lifetimes" %}
+{% set version = "0.11.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 75862d86581e75f0c235d830590bea0a9062222f2c9c390949f6432a3fa329b4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - autograd >=1.2.0
+    - dill >=0.2.6
+    - numpy >=1.10.0
+    - pandas >=0.24.0
+    - pip
+    - python
+    - scipy >=1.0.0
+    - setuptools
+  run:
+    - autograd >=1.2.0
+    - dill >=0.2.6
+    - numpy >=1.10.0
+    - pandas >=0.24.0
+    - python
+    - scipy >=1.0.0
+    - setuptools
```

Comment:
Do you need setuptools at runtime?

Reply:
Yes. A sub-package initialization depends on a function in `setuptools`. Removing this fails the test.

Reply:
Ok!